### ISSUE TITLE
spatial order of grayim vs. colorim

### ISSUE THREAD

**Issue by bjarthur** [JuliaImages/Images.jl#447]
is it by design that the spatial order returned by `grayim` and `colorim` are reversed?  this behavior is not intuitive to me.

```
julia> grayim(rand(5,4))
Gray Images.Image with:
  data: 5x4 Array{Float64,2}
  properties:
    colorspace: Gray
    spatialorder:  x y

julia> colorim(rand(5,4,3))
RGB Images.Image with:
  data: 5x4x3 Array{Float64,3}
  properties:
    colorspace: RGB
    colordim: 3
    spatialorder:  y x
```

**Comment by timholy** [comment]
I doubt it, and indeed that seems quite dreadful.

**Comment by greenflash1357** [comment]
This issue struck me when saving images to disk.

**Comment by timholy** [comment]
Closed by #577
